### ISSUE TITLE
Bug in Event default created_at time

### ISSUE THREAD

**Issue by armstrys** [jeffthibault/python-nostr#23]
The default value for created_at in the `Event` class seems to hold the `time.time()` value from the time of import.

Example of a failing test below:

```python
from nostr.event import Event
from nostr.key import PrivateKey
import time

time.sleep(2)

event = Event(public_key=PrivateKey().public_key.hex(), content='this is a test')
assert event.created_at == int(time.time())
```

**Comment by jeffthibault** [comment]
Hi @armstrys, the Event class sets the `created_at` field to the current time by default.

If I correctly understand what you are trying to do, you can do something like this:
```python
curr_time = time.time()
event = Event(public_key, "this is a test", created_at=curr_time)
assert event.created_at == curr_time
```

**Comment by armstrys** [comment]
Hi @jeffthibault! What I am noticing on my machine is that the default value is being set as soon as the module is imported as opposed to when an `Event` object is instantiated. So every event I was creating was carrying the same time stamp despite being created at different times. The `time.sleep(2)` call is creating a delay between the import and the instantiation to try to test this. 

Basically the default time isn’t being recalculated for each event properly. If I’m seeing this wrong then I agree it would make more sense to keep it defined as the arg default.

**Comment by flothjl** [comment]
@armstrys I agree with you. Didn't realize you had opened a PR so I did, but have sense deleted it. I think the problem lies in that the default args are determined on import, not initialization.

**Comment by jeffthibault** [comment]
Fixed in #24
